src/engine/resolvers.rs

```diff
@@ -80,6 +81,65 @@ where
         }
     }
 
+    /// Returns the resolver input deserialized into a structure of type T that
+    /// implements the serde `Deserialize` trait.
+    ///
+    /// # Errors
+    ///
+    /// Returns an [`Error]` variant [`InputNotFound`] if no input field was passed
+    /// to the query, [`TypeConversionFailed`] if unable to convert a warpgrapher Value
+    /// to a serde_json Value, and [`JsonDeserializationFaild`] if unable to parse the
```

Comment:
I'd recommend replacing "serde_json Value" with "[`serde_json::Value`]" or whatever the correct linked fully qualified name would be.